python/pyproject.toml

```diff
@@ -35,7 +35,7 @@ dependencies = [
 [project.optional-dependencies]
 docs = ["markdown", "pyyaml", "sphinx", "sphinx_rtd_theme"]
 lint = ["ruff"]
-optional = ["numba"]
+optional = ["numba", "pyamg"]
```

Comment:
```suggestion
optional = ["numba"]
demos = ["numba", "pyamg"]
```

Reply:
I'm just removing it for now.

Reply:
Removing it means that it will never be executed on CI?